src/coreclr/jit/lsra.cpp

```diff
@@ -11896,7 +11923,7 @@ void LinearScan::verifyFreeRegisters(regMaskTP regsToFree)
         // If this is occupied by a double interval, skip the corresponding float reg.
         if ((assignedInterval != nullptr) && (assignedInterval->registerType == TYP_DOUBLE))
         {
-            reg = REG_NEXT(reg);
+            NEXT_REGISTER(reg, regIndex);
```

Comment:
you can keep this as it is, no? For ARM, `regIndex` is not read anyway, right?

Reply:
> you can keep this as it is, no? For ARM, `regIndex` is not read anyway, right?

Yes. I updated it for the sake for uniformity. But could leave it as `reg = REG_NEXT(reg);`